packages/bundler-plugin-core/src/index.ts

```diff
@@ -58,6 +59,20 @@ export function sentryUnpluginFactory({
   debugIdUploadPlugin,
 }: SentryUnpluginFactoryOptions) {
   return createUnplugin<Options, true>((userOptions, unpluginMetaContext) => {
+    const logger = createLogger({
+      prefix: `[sentry-${unpluginMetaContext.framework}-plugin]`,
+      silent: userOptions.silent ?? false,
+      debug: userOptions.debug ?? false,
+    });
+
+    const dotenvResult = dotenv.config({
+      path: path.join(process.cwd(), ".env.sentry-build-plugin"),
+    });
+
+    if ("parsed" in dotenvResult) {
```

Comment:
l/q: What happens if this errored? Should we log something in this case? Just wondering if in an error case, some variables might have been pulled in while others weren't?...

Reply:
Good point. Added that we throw on any other error except for ENOENT!